docs/enhancements/api-for-skipping-validations.md

```diff
@@ -0,0 +1,155 @@
+* If an attempt is made to reference a validation that does not exist then a warning will be logged but this will not be considered fatal.
+* After the call has completed, if this leads to a change in the collection of skipped validations then an event will be emitted.
+
+### Edit the validations
+
+PUT api/v2/cluster/<id>/skipped-validations
+
+Essentially a document will be submitted via POST in JSON
+```
+{
+  "skipped-host-validations": [
+    "no-ip-collisions-in-network"
+  ],
+  "skipped-cluster-validations": [
+    "api-vips-defined"
+  ]
+}
+```
+* The freshly submitted JSON document, will be the new source of truth. All existing "skip operations" will be undone before applying this.
+* If an attempt is made to reference a validation that does not exist then a warning will be logged but this will not be considered fatal.
+* Submitting an empty doc '{}' will lead to the deletion of all skipped validations.
+* After the call has completed, if this leads to a change in the collection of skipped validations then an event will be emitted.
+
+### Effect on validation.
+
+When a validation is skipped then this should be reflected in the Validation status. For example;
+```
+{
+    "id": "api-vips-defined",
+    "status": "skipped",
+    "message": "API virtual IPs validation has been suppressed by the user"
+}
+```
+
+### Effect on Triage tickets.
+To ensure that there is less noise for Triage tickets, any cluster that has events consistent with this sort of override should be omitted from Triage ticketing, in a similar way to how this is done for jupiter installations.
+
+Example here: https://github.com/openshift-assisted/assisted-installer-deployment/blob/463dc6612c923c46d47e80762c72651758a690d3/tools/create_triage_tickets.py#L64
+
+### Per customer feature enablement.
+This feature is to be enabled on a per customer basis, so will need to use the "per customer feature enablement" feature.
+
+See the following link for more information.
+
+https://github.com/openshift/assisted-service/blob/master/docs/dev/feature-per-customer.md
+
+### User Stories
+
+#### Story 1
+
+As a Kaloom user, I would like to be able to bypass VIP validation in cases where VIPS could be incorrectly shown as busy. So that I may proceed with installation.
+
+"Kaloom has very specific setup where vips can be shown as busy though installation can proceed with them.
+Currently they need to override vips in install config to be able to install cluster.
+After adding the new api they can just run it and skip this specific validation."
+
+### Risks and Mitigations
+
+- We should not add uneccesary noise to triage tickets with this, it is therefore quite important to add some signatures to indicate that these validations have been bypassed.
+
+### Open Questions
+
+N/A
+
+### UI Impact
+
+As this change is API based, I am not 100% sure we would need to do too much in the UI at this stage. 
+Perhaps listing any blocked validations would be helpful?
```

Comment:
if "validations_info" has a new value "skipped", then UI might need to prepare for it (currently it expects either success or failure)

Reply:
I wonder, if it might be better to just log the validations that have been skipped instead of adding a field because

This feature is unsupported and generally we should steer away from UI indications.
Providing some indication to the end user that "some validations were skipped therefore we cannot support this installation" might be an idea, but I question how granular we have to be in this messaging

What are your thoughts?

Reply:
it's not adding a new field, it's adding a new possible value to an existing field
the problem with log lines, is that it makes it harder to understand the situation. think of a qe / dev / user that tries to understand the "history" of a validation. if it's marked as skipped it's more apparent than a log line that is not easily viewed (or not at all in some cases)

Reply:
OK, so seems like we need to add a UI task to support the new value. I'll take it into account.